ext/mysqli/tests/mysqli_connect_oo.phpt

```diff
@@ -66,21 +66,19 @@ require_once('skipifconnectfailure.inc');
         }
     }
 
-    if ($IS_MYSQLND) {
-        ini_set('mysqli.default_host', 'p:' . $host);
-        if (!is_object($mysqli = new mysqli())) {
-            // Due to an API flaw this shall not connect
-            printf("[010] Failed to create mysqli object\n");
-        } else {
-            // There shall be NO connection! Using new mysqli(void) shall not use defaults for a connection!
-            // We had long discussions on this and found that the ext/mysqli API as
-            // such is broken. As we can't fix it, we document how it has behaved from
-            // the first day on. And that's: no connection.
-            try {
-                $mysqli->query('SELECT 1');
-            } catch (Error $exception) {
-                echo $exception->getMessage() . "\n";
-            }
+    ini_set('mysqli.default_host', 'p:' . $host);
+    if (!is_object($mysqli = new mysqli())) {
```

Comment:
Unrelated, but this if can be dropped...

Reply:
@nikic true, but it can also stay; I mean its harmless.

Reply:
> I mean its harmless

Yeah, but confusing, and that leads to code like

https://github.com/php/php-src/blob/ed652a514fab0cbe797504f850ea64bd2ed1c4d2/ext/mysqli/tests/mysqli_class_mysqli_result_interface.phpt#L113-L114

There is no `gettye()` function.

> Why there is a blank space between last `)` and `{` (line 70 of above snippet)?

That's our preferred coding style.

Reply:
Ugh, there were several such useless checks. Removed with 2da68903a6579b78bc2d6c5ef0d5bbb020280da4.